tests/unit/test_sampler.py

```
@@ -0,0 +1,209 @@
+            assert 20 <= age <= 30, f"Age {age} not in range 20-30"
+    
+    @pytest.mark.asyncio
+    async def test_sample_with_gender_filter(self, data_paths):
+        """Test sampling with gender filter."""
+        # Sample only female individuals
+        samples = await sample(
+            qty=5,
+            gender="F",
+            make_api_call=False,
+            **data_paths
+        )
```

Ruff (0.12.2)

Use of `assert` detected

(S101)